LFGBulletinBoard/LibGPIToolBox.lua

```diff
@@ -1,8 +1,10 @@
 local TOCNAME,
 	---@class Addon_Tool	
 	Addon = ...;
-Addon.Tool=Addon.Tool or {}
```

Comment:
`Addon.Tool` is not defined elsewhere and this code only runs once.
Equivalent to saying `Addon.Tool = {}`